scripts/kind-with-registry.sh

```diff
@@ -42,37 +56,140 @@ if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true
     registry:2
 fi
 
-# To use workload identity, service account signing key pairs base64 encoded should be exposed via the
-# env variables. The function creates the key pair files after reading it from the env variables.
+# Environment variable inputs
+# SERVICE_ACCOUNT_ISSUER - BYO existing service account issuer
+#    Accepts a URI string, e.g., https://${AZWI_STORAGE_ACCOUNT}.blob.core.windows.net/${AZWI_STORAGE_CONTAINER}/
+#    Assumes that the required openid and jwks artifacts exist in the well-known locations at this URI
+# SERVICE_ACCOUNT_SIGNING_PUB_FILEPATH - A local filepath to a 2048 bit RSA public key
+#    Defaults to capz-wi-sa.pub, must exist locally and match the signed artifacts if using BYO
+# SERVICE_ACCOUNT_SIGNING_KEY_FILEPATH - A local filepath to a 2048 bit RSA private key
+#    Defaults to capz-wi-sa.key, must exist locally and match the signed artifacts if using BYO
+#    If the above keypair filepaths environment variables are not included, a keypair will be created at runtime
+#    Note: if a new keypair is created at runtime then you must not BYO service account issuer
+# AZWI_RESOURCE_GROUP - Azure resource group where Workload Identity infra lives
+# AZWI_LOCATION - Azure location for Workload Identity infra
+# AZWI_STORAGE_ACCOUNT - Storage account in resource group $AZWI_RESOURCE_GROUP containing required artifacts
+# AZWI_STORAGE_CONTAINER - Name to use when creating a storage container in storage account $AZWI_STORAGE_ACCOUNT
+# AZURE_CLIENT_ID_USER_ASSIGNED_IDENTITY - BYO existing user-assigned identity
+#    Should be a UUID that represents the clientID of the identity object
+# USER_IDENTITY - Name to use when creating a new user-assigned identity
+#    Required if not bringing your own identity via $AZURE_CLIENT_ID_USER_ASSIGNED_IDENTITY
+# AZURE_IDENTITY_ID_FILEPATH - A local filepath to store the newly created user-assigned identity if not bringing your own
```

Comment:
This is for a file, not a filepath so it's confusing.  I'd also say to just default this to a value since the name is not important.

Reply:
Good call, defaulted to a value.